pyani_plus/methods/method_fastani.py

```diff
@@ -23,6 +23,11 @@
 
 from pathlib import Path
 
+KMER_SIZE = 16  # Default fastANI k-mer size (max 16)
```

Comment:
At some point I'd like us to move constants like these into a single default config file for the repo/package, and to allow a user to define their own settings in a local config file to override them.

Reply:
OK, although there is something to be said for method specific constants in each method file too.

Handling overrides to command line options is quite a common UI pattern, although there is more than one way to implement it. So that does make sense as a future addition 👍